### PR TITLE
Update source to produce 1.0 data

### PR DESCRIPTION
**Pull request by matzew**
## Proposed Changes

  * have our source now produce 1.0 compliant data

## Github source

@bbrowning @lberk @lionelvillard  For GH Source I had some funky change to do.... 

see https://github.com/knative/eventing-contrib/issues/780  

**Release Note**

```release-note
Due to cloudEvent spec changes, the github delivery and github event EXTENSION was renamed to just event and delivery
```

### CODE REVIEW

File: kafka/source/pkg/adapter/adapter.go [hunk 1]
```diff
@@ -181,6 +181,7 @@ func (a *Adapter) Handle(ctx context.Context, msg *sarama.ConsumerMessage) (bool
 	event.SetTime(msg.Timestamp)
 	event.SetType(sourcesv1alpha1.KafkaEventType)
 	event.SetSource(sourcesv1alpha1.KafkaEventSource(a.config.Namespace, a.config.Name, msg.Topic))
+	event.SetSubject(sourcesv1alpha1.KafkaEventSubject(msg.Partition, msg.Offset))
```
Comment (lionelvillard) [inline kafka/source/pkg/adapter/adapter.go]: shouldn't be here?

File: kafka/source/pkg/apis/sources/v1alpha1/kafka_types.go [hunk 2]
```diff
@@ -135,6 +135,11 @@ func KafkaEventSource(namespace, kafkaSourceName, topic string) string {
 	return fmt.Sprintf("/apis/v1/namespaces/%s/kafkasources/%s#%s", namespace, kafkaSourceName, topic)
 }
 
+// KafkaEventSource returns the Kafka CloudEvent subject of the message.
```
Comment (lionelvillard) [inline kafka/source/pkg/apis/sources/v1alpha1/kafka_types.go]: shouldn't be here?

File: awssqs/pkg/adapter/adapter.go [hunk 3]
```diff
@@ -179,12 +179,12 @@ func (a *Adapter) postMessage(ctx context.Context, logger *zap.SugaredLogger, m
 	}
 
 	event := cloudevents.Event{
```
Comment (n3wscott) [inline awssqs/pkg/adapter/adapter.go]: Use `cloudevents.NewEvent("1.0")`

File: awssqs/pkg/adapter/adapter.go [hunk 4]
```diff
@@ -179,12 +179,12 @@ func (a *Adapter) postMessage(ctx context.Context, logger *zap.SugaredLogger, m
...
-		Context: cloudevents.EventContextV03{
+		Context: cloudevents.EventContextV1{
 			ID:     *m.MessageId,
```
Comment (n3wscott) [inline awssqs/pkg/adapter/adapter.go]: And then you can use the setters with the go native types.

File: github/pkg/adapter/adapter.go [hunk 5]
```diff
@@ -34,6 +34,8 @@ import (
 const (
 	GHHeaderEvent    = "GitHub-Event"
 	GHHeaderDelivery = "GitHub-Delivery"
+	GHCEHeaderEvent    = "event"
```
Comment (n3wscott) [inline github/pkg/adapter/adapter.go]: You could take a look at https://github.com/cloudevents/spec/blob/master/adapters/github.md and see if you agree with @duglin

File: kafka/source/pkg/apis/sources/v1alpha1/kafka_types_test.go [hunk 6]
```diff
@@ -27,4 +27,4 @@ func TestKafkaSource_GetGroupVersionKind(t *testing.T) {
 	if gvk.Kind != "KafkaSource" {
 		t.Errorf("Should be 'KafkaSource'.")
 	}
-}
+}
```
Comment (mattmoor) [inline kafka/source/pkg/apis/sources/v1alpha1/kafka_types_test.go]: Add trailing newline:
```suggestion
}

```

Comment (mattmoor) [inline kafka/source/pkg/apis/sources/v1alpha1/kafka_types_test.go]: Add trailing newline:
```suggestion
}

```

**Pull request merged**